dpbench/infrastructure/frameworks/framework.py

```diff
@@ -41,6 +41,11 @@ def __init__(
         self.info = config
         self.fname = self.info.simple_name
 
+        import dpctl
+
+        # TODO: find better way to get cpu info
+        self.device_info = dpctl.SyclDevice("opencl:cpu").name
```

Comment:
```suggestion
import cpuinfo

self.device_info = cpuinfo.get_cpu_info().get("brand_raw")
```

We can use `py-cpuinfo`

Reply:
Yeah, I agree it is a better way. Just did not want to introduce new dependency, but I guess it's okay.